src/containers/teleport/pages/mainScreen/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+.SwapActionContentContainer {
+  display: grid;
```

Comment:
- split to components folders
- use lowercase first letter
- more simple names (now is long)